README.md

```diff
@@ -2,7 +2,7 @@
 
 # Optuna: A hyperparameter optimization framework
 
-[![Python](https://img.shields.io/badge/python-3.5%20%7C%203.6%20%7C%203.7%20%7C%203.8-blue)](https://www.python.org)
+[![Python](https://img.shields.io/pypi/pyversions/optuna)](https://www.python.org)
```

Comment:
This badge automatically extracts supported python versions from PyPI:

![Screen Shot 2020-09-12 at 14 14 50](https://user-images.githubusercontent.com/17039389/92987968-64512280-f502-11ea-877f-b718a81c610f.png)

https://github.com/harupy/optuna/tree/update-py-version#optuna-a-hyperparameter-optimization-framework

Reply:
I didn't know the feature. It seems very convenient.